### DISCUSSION TITLE
Week 11- Project Updates

### DISCUSSION THREAD

**Discussion by CJ3315** [USU-ITLS-x265-23-summer/class-repo#203] [General]
**Tasks Completed This Week:**

**About me section:**

- Grid created 
- box shadows added 
- Borders created 
- Text added 
- Skills list added including progress bars and percentages 
- Career Timeline added 

**Youtube Section**

- 9 videos added with working links and transitions effects 
- Icons added for play buttons 
- Video titles added 
- Thumbnail Images added 

**Photoshop Section**

- 6 exhibits added with transition effects 
- Descriptions and titles added 
- Images added 
- Backgrounds added 

**Tasks to be Completed **

- Information on the contact page 
- Interactive buttons on the page 
- Refine styles and fonts 
- Touch-up sections 
- Add more text to Youtube page 
- Maybe add images on the about me page

**Comment by CJ3315** [comment]
Also here is a link to my project page: https://github.com/CJ3315/Portofolio-Project

**Reply by audreyhki** [reply]
@CJ3315 You've done a great job so far showing your competency in the programs we've learned this semester. Your project and task lists are definitely what I use for inspiration and ideas of things that may be missing. Keep up the good work, I can't wait to see your final project!

**Reply by CJ3315** [reply]
Thank you for the kind words! How is your project going?

**Reply by audreyhki** [reply]
It's going well! I'm excited for the semester to be over and have a two-week break before the fall semester begins!

**Comment by beto0890** [comment]
Great progress. I am also considering using some shadows in the cards, let me know if you run into problems with them it took me a while how to do it when it was kinda simple.
